var/spack/repos/builtin/packages/py-gql/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+class PyGql(PythonPackage):
+    """This is a GraphQL client for Python. Plays nicely with
+    graphene, graphql-core, graphql-js and any other GraphQL
+    implementation compatible with the spec.
+    GQL architecture is inspired by React-Relay and Apollo-Client."""
+
+    homepage = "https://github.com/graphql-python/gql"
+    url      = "https://github.com/graphql-python/gql/archive/v3.0.0a1.tar.gz"
+
+    version('3.0.0a1', sha256='3254a6010464932e3700a8d225cf6e40a6983aaf5f279615504c8196a374daf9')
```

Comment:
It looks like this is a pre-release version. You can add it if you want, but it would be better to add the latest stable release instead. If you want both, add `preferred=True` to the latest stable release.

Reply:
I'd rather start at 3.0.0 (even if it is alpha at this point) instead of dragging the "python2 version" into existence.

Reply:
I'm fine with that, we can remove the alpha release once a stable release of 3.0.0 comes out.